engine/src/main/java/org/destinationsol/game/BeaconHandler.java

```diff
@@ -183,10 +183,10 @@ private void updateD(SolGame game) {
                 farDrawable = (FarDrawable) fo;
                 return;
             }
-            throw new AssertionError();
+            throw new AssertionError("The far object data in game is a FarDrawable object, does not contain 3 elements in drawables, or the first drawable isnt attackSprite");
         }
         if (farDrawable == null) {
-            throw new AssertionError("Far drawable does not exist!");
+            throw new AssertionError("Far drawable is null!");
```

Comment:
This is basically just a rewording - "is null" is no clearer than "does not exist".